### PR TITLE
Simplify mysql tests

### PR DESCRIPTION
**Pull request by aminalaee**
An internal change to simplify the mysql tests now that Python 3.6 does not need to be supported.
Before aiomysql didn't support Py3.10 and asyncmy didn't support Py3.6, now both these limitations are removed.

### CODE REVIEW

**Comment by aminalaee** [comment]
Need to wait for aiomysql to be released here: https://github.com/aio-libs/aiomysql/pull/645

**Comment by Kludex** [comment]
There's another conditional there that was not explained on the description 😟

**Comment by aminalaee** [comment]
@Kludex I didn't get what you mean.

File: tests/test_databases.py [hunk 1]
```diff
@@ -17,23 +16,6 @@
...
-def mysql_versions(wrapped_func):
-    """
-    Decorator used to handle multiple versions of Python for mysql drivers
-    """
-
-    @functools.wraps(wrapped_func)
-    def check(*args, **kwargs):  # pragma: no cover
-        url = DatabaseURL(kwargs["database_url"])
-        if url.scheme in ["mysql", "mysql+aiomysql"] and sys.version_info >= (3, 10):
-            pytest.skip("aiomysql supports python 3.9 and lower")
```
Comment (Kludex) [inline tests/test_databases.py:29]: The description of this PR only mentions Python 3.6, but this conditional also exists there.

Comment (aminalaee) [reply tests/test_databases.py:29]: Yeah aiomysql now supports Py3.10. I'll add it there.

Comment (aminalaee) [reply tests/test_databases.py:29]: Done.

**Comment by aminalaee** [comment]
@Kludex Thank you.

**Review by Kludex — APPROVED** [review]

**Pull request merged**